tests/spirv/tess-factor.slang

```diff
@@ -0,0 +1,32 @@
+//TEST:SIMPLE(filecheck=CHECK): -target spirv -stage domain
```

Comment:
should we add a glsl test for `gl_TessLevelInner`?

Reply:
We should, but that can come in a separate PR.